wwwroot/js/site.js

```diff
@@ -2,3 +2,71 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+
+/* Selectable table row functionality */
+$("table.table-selectable tbody tr").click(function () {
+    $("table.table-selectable tbody tr").removeClass('selectedRow');
+    $(this).addClass('selectedRow');
+});
+
+
+/* Filters a searchable table based on search-input */
+$(document).ready(function () {
+    $("#search-input").on("keyup", function () {
+        var value = $(this).val().toLowerCase();
+        $("table.table-searchable tbody tr").filter(function () {
+            $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
+        });
+    });
+});
+
+
+/* Load PersonDetailsPartial async */
+function loadPersonDetailsPartial(id) {
```

Comment:
should functions, that are only intended to be used in one view, be in the main JS? I think it should be split up, but not much of a problem for now